### PR TITLE
CID-2719 - Changes to update user MFA settings to support RACKSPACE_MANDATED

### PR DESCRIPTION
**Pull request by abhatt1250**

### CODE REVIEW

**Comment by rackernexus** [comment]
Can one of the admins verify this patch?

File: api-docs/api-reference/methods/multifactor-auth/put-update-multifactor-settings-on-account-v2.0.rst [hunk 1]
```diff
@@ -19,6 +19,12 @@ on an account.  Note that this feature is available only through the API. You
 cannot override domain enforcement settings from the Rackspace Cloud Control
 Panel.
 
+.. note::
+
+ - Customers need a support ticket if they want to update user's MFA
```
Comment (catlook) [inline api-docs/api-reference/methods/multifactor-auth/put-update-multifactor-settings-on-account-v2.0.rst:24]: ... update a user's ...

**Review by catlook — CHANGES_REQUESTED** [review]
one tweak

File: api-docs/api-reference/methods/multifactor-auth/put-update-multifactor-settings-on-account-v2.0.rst (hunk 1 above)
Comment (abhatt1250) [reply api-docs/api-reference/methods/multifactor-auth/put-update-multifactor-settings-on-account-v2.0.rst]: Addressed

**Review by catlook — APPROVED** [review]
👍

**Review by premaseem — APPROVED** [review]

**Review by nonameentername — APPROVED** [review]

**Pull request merged**